lib/common.bash

```diff
@@ -281,3 +281,36 @@ restart_docker_service() {
 
 	return 0
 }
+
+# This function verifies that the dockerfile version is
+# equal to the test version in order to build the image or
+# just run the test
+check_dockerfiles_images()
+{
+	local image="$1"
+	local dockerfile_path="$2"
+
+	if [ -z "$image" ] || [ -z "$dockerfile_path" ]; then
+		die "Missing image or dockerfile path variable"
+	fi
+
+	# Verify that dockerfile version is equal to test version
+	check_image=$(docker images "$image" -q)
```

Comment:
Hi @GabyCT , I know nothing about the metrics CI but let me try to help with review of this...

`docker images "$image" -q` can return more than one hash if the "$image" hasn't a specific tag. For example,

`
[wmoschet@wainer-laptop kubernetes]$ docker images "docker.io/library/ubuntu"
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
REPOSITORY                TAG         IMAGE ID      CREATED        SIZE
docker.io/library/ubuntu  18.04       2c047404e52d  11 months ago  65.6 MB
docker.io/library/ubuntu  20.04       d70eaf7277ea  12 months ago  75.3 MB
`

Reply:
@wainersm yes it retrieves several images but then it looks for a particular image with a specific tag